samples/AndroidSample/MainActivity.cs

```diff
@@ -18,13 +18,6 @@
 using SampleTools;
 using AndroidHUD;
 
-// This will add your license key into AndroidManifest.xml at build time. For more info on how this Attribute works see:
-// https://developer.xamarin.com/guides/android/advanced_topics/working_with_androidmanifest.xml/
-//[assembly: MetaData (
-//	name:"pspdfkit_license_key",
-//	Value = "LICENSE_KEY_GOES_HERE"
-//)]
```

Comment:
@dalexsoto Is there any reason to leave this in? If used the hybrids platform will not be set so licensing should fail.

Reply:
@matej yep this is why I am getting rid of it in this PR, from now on the Initialize call is the way to go